saphanabootstrap-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb 11 10:51:56 UTC 2020 - Dario Maiocchi <dmaiocchi@suse.com>
+
+-  Version 0.4.5
+  * Change name of pkg to prometheus-hanadb_exporter (jsc#SLE-10545)
```

Comment:
I would rewrite the message to

`Change the package name to prometheus-hanadb_exporter (jsc#SLE-10545)`
I would prefer to not use SMS kind of texts if we at least find them during the review.